Docs/update PURIS KIT 25.09 (#1333)
eclipse-tractusx/eclipse-tractusx.github.io: docs-kits/kits/puris-kit/adoption-view/integration-scenarios/multi-sourcing-scenario.md

---
id: Multi Sourcing Scenario
title: Multi Sourcing Scenario
description: Multi Sourcing Scenario
sidebar_position: 5
---

In a Multi-Sourcing scenario, the same material or product is procured from multiple suppliers. While the internal handling of the material—such as storage or production use—may be unified, companies must ensure that data related to demand and item stock can be clearly attributed to the respective supplier.

In cases where supplier-specific information is maintained separately (e.g., through batch tracking or source-specific labeling), data provisioning is relatively straightforward. The origin of materials can be identified and shared without major transformation.

![customer-multi-sourcing-part1](../../assets/customer-multi-sourcing-part1.svg)

![customer-multi-sourcing-part2](../../assets/customer-multi-sourcing-part2.svg)

Figure 1 and 2: *Customer multi sourcing scenarios*

> In practice, this means that demand planning and inventory management are handled jointly, making it infeasible to assign data to a specific supplier context. Nevertheless, the data must still be prepared and structured in a way that complies with applicable legal, regulatory, and contractual requirements—even if a clear technical separation is not possible.

Reviewer: ```suggestion
In practice, this means that demand planning and inventory management are handled jointly, making it infeasible to assign data to a specific supplier context. Nevertheless, it is recommended to prepare data in a structured way that complies with applicable legal, regulatory, and contractual requirements—even if a clear technical separation is not possible.
```
based on what do we say "data must be still prepared" ? in todays business scenerio, this is not a must have this is part of business contracts negotiated and agreed... can not be forced as an industry standard, should not be "must", to be changed to "recommendation"

Reviewer 2: Referring to section 5.2.3 of the CX-0120 standard, it is stated that "participants must comply with legal restrictions, such as competition laws" and that "any Short-Term Material Demand provided must be partner-specific." These formulations indicate that certain requirements—particularly regarding data handling in multi-sourcing scenarios—are not merely best practices but essential conditions for using the standard appropriately.

In this context, the use of the term "recommended" may not fully reflect the binding nature of these obligations. A more precise wording could help ensure that the importance of legal compliance and partner-specific data provisioning is clearly understood and consistently applied.
